docs/tutorials/zarr/chunked.md

```diff
@@ -78,13 +155,16 @@ group = zarr.open(config.video[0].storage.settings.filename)
 assert len(group) == 1
 
 # inspect the characteristics of the data
-group["0"]
+print(group["0"])
 ```
 
 The output will be:
 ```
-<zarr.core.Array '/0' (10, 1, 1080, 1920) uint8>
+<zarr.core.Array '/0' (300, 3, 10, 1080, 1920) uint8>
 ```
-As expected, we have only 1 top level directory, corresponding to the single array in the group. We would expect more than 1 array only if we were writing [multiscale data](multiscale.md). The overall array shape is (10, 1, 1080, 1920), corresponding to 10 frames, 1 channel, and a height and width of 1080 and 1920, respectively, per frame.
+As expected, we have only 1 top level directory, corresponding to the single array in the group.
+We would expect more than 1 array only if we were writing [multiscale data](multiscale.md).
+The overall array shape is (10, 1, 1080, 1920), corresponding to 10 frames, 1 channel, and a height and width of 1080
```

Comment:
Shape is wrong here. The 300 in line 163 is a bug -- it should be 10 (= append dimension divided by internal dimensions, i.e., 300 / 3 / 10). If the bug fix could only make it through review, the shape you'd expect in this line is (10, 3, 10, 1080, 1920) corresponding to
- 10 time points
- 3 channels
- 10 planes
- 1080 rows
- 1920 columns

Reply:
Thanks for the explanation!

This tutorial got added back when Melissa rebased. I will delete again now as discussed and we can add it back when the bug is fixed.